### ISSUE TITLE
Installation fails on Node 19.4.0

### ISSUE THREAD

**Issue by Cykelero** [keichi/osx-tag#14]
I'm trying to use osx-tag on an Apple Silicon Mac, using Node 19.2.0 with npm 9.2.0. `npm install` outputs this error:

```
npm ERR! code 1
npm ERR! path /Users/cykelero/.cache/tasklemon/npm-packages/11398b761b8b007410af4996bab7eb0cd44d1c94990e93022109686e5a0e78b9/node_modules/osx-tag
npm ERR! command failed
npm ERR! command sh -c node-gyp rebuild
npm ERR! CXX(target) Release/obj.target/osx_tag/osx_tag.o
npm ERR! gyp info it worked if it ends with ok
npm ERR! gyp info using node-gyp@9.3.0
npm ERR! gyp info using node@19.4.0 | darwin | arm64
npm ERR! gyp info find Python using Python version 3.10.9 found at "/opt/homebrew/opt/python@3.10/bin/python3.10"
npm ERR! gyp info spawn /opt/homebrew/opt/python@3.10/bin/python3.10
npm ERR! gyp info spawn args [
npm ERR! gyp info spawn args   '/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py',
npm ERR! gyp info spawn args   'binding.gyp',
npm ERR! gyp info spawn args   '-f',
npm ERR! gyp info spawn args   'make',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/cykelero/.cache/tasklemon/npm-packages/11398b761b8b007410af4996bab7eb0cd44d1c94990e93022109686e5a0e78b9/node_modules/osx-tag/build/config.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/addon.gypi',
npm ERR! gyp info spawn args   '-I',
npm ERR! gyp info spawn args   '/Users/cykelero/Library/Caches/node-gyp/19.4.0/include/node/common.gypi',
npm ERR! gyp info spawn args   '-Dlibrary=shared_library',
npm ERR! gyp info spawn args   '-Dvisibility=default',
npm ERR! gyp info spawn args   '-Dnode_root_dir=/Users/cykelero/Library/Caches/node-gyp/19.4.0',
npm ERR! gyp info spawn args   '-Dnode_gyp_dir=/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp',
npm ERR! gyp info spawn args   '-Dnode_lib_file=/Users/cykelero/Library/Caches/node-gyp/19.4.0/<(target_arch)/node.lib',
npm ERR! gyp info spawn args   '-Dmodule_root_dir=/Users/cykelero/.cache/tasklemon/npm-packages/11398b761b8b007410af4996bab7eb0cd44d1c94990e93022109686e5a0e78b9/node_modules/osx-tag',
npm ERR! gyp info spawn args   '-Dnode_engine=v8',
npm ERR! gyp info spawn args   '--depth=.',
npm ERR! gyp info spawn args   '--no-parallel',
npm ERR! gyp info spawn args   '--generator-output',
npm ERR! gyp info spawn args   'build',
npm ERR! gyp info spawn args   '-Goutput_dir=.'
npm ERR! gyp info spawn args ]
npm ERR! gyp info spawn make
npm ERR! gyp info spawn args [ 'BUILDTYPE=Release', '-C', 'build' ]
npm ERR! In file included from ../osx_tag.mm:3:
npm ERR! In file included from ../../nan/nan.h:180:
npm ERR! ../../nan/nan_callbacks.h:55:23: error: no member named 'AccessorSignature' in namespace 'v8'
npm ERR! typedef v8::Local<v8::AccessorSignature> Sig;
npm ERR!                   ~~~~^
npm ERR! 1 error generated.
npm ERR! make: *** [Release/obj.target/osx_tag/osx_tag.o] Error 1
npm ERR! gyp ERR! build error 
npm ERR! gyp ERR! stack Error: `make` failed with exit code: 2
npm ERR! gyp ERR! stack     at ChildProcess.onExit (/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/lib/build.js:203:23)
npm ERR! gyp ERR! stack     at ChildProcess.emit (node:events:513:28)
npm ERR! gyp ERR! stack     at ChildProcess._handle.onexit (node:internal/child_process:293:12)
npm ERR! gyp ERR! System Darwin 22.2.0
npm ERR! gyp ERR! command "/opt/homebrew/Cellar/node/19.4.0/bin/node" "/opt/homebrew/lib/node_modules/npm/node_modules/node-gyp/bin/node-gyp.js" "rebuild"
npm ERR! gyp ERR! cwd /Users/cykelero/.cache/tasklemon/npm-packages/11398b761b8b007410af4996bab7eb0cd44d1c94990e93022109686e5a0e78b9/node_modules/osx-tag
npm ERR! gyp ERR! node -v v19.4.0
npm ERR! gyp ERR! node-gyp -v v9.3.0
npm ERR! gyp ERR! not ok
24414   │ 2023-01-18T17:02:07.380Z 
npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/cykelero/.npm/_logs/2023-01-18T17_02_05_302Z-debug-0.log
`npm install` failed (code 1).
```

Is there something I can do to fix this?

**Comment by keichi** [comment]
Hi, I just published a new version of the package (0.4.7) that fixes this error. Please try it out.

**Comment by Cykelero** [comment]
It works now—thank you very much for the quick fix!
I love using osx-tag in my personal automation scripts :)
